test/drenv/__init__.py

```diff
@@ -45,7 +45,7 @@ def wait_for(
         out = kubectl.get(*args, context=profile)
         if out:
             elapsed = time.monotonic() - start
-            log(f"{resource} outuput={output} found in {elapsed:.2f} seconds")
+            log(f"{resource!r} outuput={output!r} found in {elapsed:.2f} seconds")
```

Comment:
Typo - outuput

Reply:
This is an existing issue, I added another commit fixing this and other spelling errors in this file.